decidim-core/config/locales/ca.yml

```diff
@@ -31,6 +34,17 @@ ca:
       image_too_big: La imatge és massa gran
   decidim:
     account:
+      delete:
+        alert: Aquesta acció no es pot desfer. Si elimina el seu compte no seràs capaç d'iniciar sessió.
+        confirm:
+          close: Tanca la finestra
+          ok: Sí, vull eliminar el meu compte
+          question: Estàs segur que vols eliminar el teu compte?
+          title: Eliminar el meu compte
+        explanation: Si us plau, omple la raó per la qual vols eliminar el teu compte (opcional).
+      destroy:
+        error: Hi ha hagut un error en eliminar el teu compte.
```

Comment:
Nope, this is the correct version!

http://catalallengua.blogspot.com.es/2014/10/alen-infinitiu-optimot-i-esadir.html